### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by DarjaLeonova**

### CODE REVIEW

**Review by mvilks — APPROVED** [review]

File: pkg/convert/po.go [hunk 1]
```diff
@@ -245,9 +245,9 @@ func ToPo(t model.Translation) ([]byte, error) {
 			switch p := p.(type) {
 			case ast.TextPattern:
 				text += string(p)
-			case ast.PlaceholderPattern:
+			case ast.Pattern:
 				//nolint:forcetypeassert // No other types are possible.
-				text += placeholders[p.Expression.(ast.VariableExpression).Variable]
+				text += placeholders[p.(ast.VariableExpression).Variable]
```
Comment (VladislavsPerkanuks) [inline pkg/convert/po.go:250]: What's the point of asserting interface type to the interface? I am curious why golang allows doing that :)

```go
case ast.VariableExpression: // No other types are possible for now
  text += placeholders[p.Variable]
```

File: pkg/fuzzy/google_translate.go [hunk 2]
```diff
@@ -271,7 +271,7 @@ func patternToString(pattern []ast.Pattern) string {
 		switch v := pattern[i].(type) {
 		case ast.TextPattern:
 			text += string(v)
-		case ast.PlaceholderPattern:
+		case ast.Pattern:
 			text += fmt.Sprintf("{$%d}", i)
```
Comment (VladislavsPerkanuks) [inline pkg/fuzzy/google_translate.go:275]: Same here
```suggestion
		case ast.LiteralExpression, ast.AnnotationExpression, ast.VariableExpression, ast.Markup: // or default
			text += fmt.Sprintf("{$%d}", i)
		}
```

Comment (VladislavsPerkanuks) [reply pkg/fuzzy/google_translate.go:275]: Update godoc, there is no more `PlaceholderPattern`.
Patterns are:
- Text
- All Expressions (Literal, Variable, Annotation)
- Markup

File: go.sum
Comment (VladislavsPerkanuks) [inline go.sum:1]: Something is off here.
Not the first time I seen this, in your PR's

When you work there are many additions to `go.sum` file, but when I run `go mod tidy` all those lines get deleted.
So this file always gets changed back and forth. 

So would be nice, if some 3rd party could run `go mod tidy`, and see which behavior is correct. And investigate why this inconsistency is appearing, would be nice too

**Review by VladislavsPerkanuks — APPROVED** [review]

**Review by klavinsjanis — APPROVED** [review]

File: pkg/fuzzy/google_translate.go [hunk 3]
```diff
@@ -271,7 +271,7 @@ func patternToString(pattern []ast.Pattern) string {
 		switch v := pattern[i].(type) {
 		case ast.TextPattern:
 			text += string(v)
-		case ast.PlaceholderPattern:
+		case ast.LiteralExpression, ast.AnnotationExpression, ast.VariableExpression, ast.Markup: // or default
```
Comment (VladislavsPerkanuks) [inline pkg/fuzzy/google_translate.go:274]: that default comment was for you :), shouldnt be here
either
1. case ast.LiteralExpression, ast.Annotation...
2. default

**Review by VladislavsPerkanuks — APPROVED** [review]

**Pull request merged**